src/web/components/MostViewed/MostViewedFooter/SecondTierItem.tsx

```diff
@@ -98,7 +98,12 @@ export const SecondTierItem = ({
 	} = trail;
 
 	const avatarToShow = avatarUrl || image;
-	const pilarToUse = design === Design.Comment ? Pillar.Opinion : pillar;
+	const pillarToUse = design === Design.Comment ? Pillar.Opinion : pillar;
+	const formatToUse: Format = {
+		design,
+		display: Display.Standard,
```

Comment:
The `Card` data we're sent from CAPI includes nothing about `display` so we default to `Standard`